### ISSUE TITLE
Regarding the pre-trained model performance

### ISSUE THREAD

**Issue by yanagi-lmd** [ZENGYIMING-EAMON/CorrNet3D#7]
I trained the CorrNet3D model following the instruction, however, I could not reproduce the performance of the provided pre-trained model. May I ask you to provide the best seed value or learning curve, if possible?

**Comment by AmmonZ** [comment]
Feel free to check the [original log record files](https://drive.google.com/drive/folders/1kLMnVmQ-iuSWYU8ATlIE055gRRWnq266?usp=sharing) produced by Pytorch Lightning automatically, and you can check all loss curves by the tensorboard. But since the tensorboard can't be edited, so you may extract the loss curve data from it if you need it (I haven't tried that).
Unfortunately, the seed is [generated randomly by PyTorch Lightning](https://github.com/ZENGYIMING-EAMON/CorrNet3D/blob/main/lit_corrnet3d_clean.py) (see L428) and only printed on the screen, so there is no record. Feel free to ask if you have further questions.

**Comment by ZENGYIMING-EAMON** [comment]
Feel free to check the [original log record files](https://drive.google.com/drive/folders/1kLMnVmQ-iuSWYU8ATlIE055gRRWnq266?usp=sharing) produced by Pytorch Lightning automatically, and you can check all loss curves by the tensorboard. But since the tensorboard can't be edited, so you may extract the loss curve data from it if you need it (I haven't tried that).
Unfortunately, the seed is [generated randomly by PyTorch Lightning](https://github.com/ZENGYIMING-EAMON/CorrNet3D/blob/main/lit_corrnet3d_clean.py) (see L428) and only printed on the screen, so there is no record. Feel free to ask if you have further questions.
